### PR TITLE
Add Migration procedure for upgrading MySQL v5.5->5.7

### PR DESCRIPTION
**Pull request by SwatiNEC**
Add migration script and procedure document to resolve [#6402](https://github.com/wso2/product-is/issues/6402)

### CODE REVIEW

File: en/docs/setup/wso2is_script.sql [hunk 1]
```diff
@@ -0,0 +1,83 @@
+ALTER TABLE `apimgt`.`IDN_OAUTH2_ACCESS_TOKEN` CHANGE COLUMN `TIME_CREATED` `TIME_CREATED` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP , CHANGE COLUMN `REFRESH_TOKEN_TIME_CREATED` `REFRESH_TOKEN_TIME_CREATED` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ;
```
Comment (JKAUSHALYA) [inline en/docs/setup/wso2is_script.sql:1]: Please remove the DB (`apimgt`) name from the query. DB name is variable.

File: en/docs/setup/wso2is_script.sql [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+
+ALTER TABLE `apimgt`.`AM_BLOCK_CONDITIONS` CHANGE COLUMN `UUID` `UUID` VARCHAR(256) NULL DEFAULT NULL ;
+
+create index IDX_AAI_CTX on apimgt.AM_API (CONTEXT);
+
+create index IDX_AAUM_TT on apimgt.AM_API_URL_MAPPING (THROTTLING_TIER);
+
+
+ALTER TABLE `regdb`.`REG_LOG` DROP INDEX `REG_LOG_IND_BY_REG_LOGTIME` , ADD INDEX `REG_LOG_IND_BY_REG_LOG` USING HASH (`REG_LOGGED_TIME`, `REG_TENANT_ID`);
```
Comment (JKAUSHALYA) [inline en/docs/setup/wso2is_script.sql:54]: I think we should divide this into multiple scripts based on the DB. So each can execute in the respective DB. And in the instructions section, we need to inform that each should execute in the correct DB.

File: en/docs/setup/wso2is_script.sql [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+ALTER TABLE `registry`.`REG_RESOURCE` CHANGE COLUMN `REG_CREATED_TIME` `REG_CREATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP , CHANGE COLUMN `REG_LAST_UPDATED_TIME` `REG_LAST_UPDATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ;
+
+ALTER TABLE `registry`.`REG_RESOURCE` DROP INDEX `REG_RESOURCE_IND_BY_TENANT` , ADD INDEX `REG_RESOURCE_IND_BY_TENAN` USING HASH (`REG_TENANT_ID`, `REG_UUID`);
+
+ALTER TABLE `registry`.`REG_RESOURCE_HISTORY` CHANGE COLUMN `REG_CREATED_TIME` `REG_CREATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP , CHANGE COLUMN `REG_LAST_UPDATED_TIME` `REG_LAST_UPDATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ;
+
+
+ALTER TABLE `apimgt`.`IDN_OAUTH2_ACCESS_TOKEN` DROP INDEX `IDX_AT` , ADD INDEX `IDX_AT` (`ACCESS_TOKEN`(250) ASC), DROP INDEX `IDX_IOAT_AT` , ADD INDEX `IDX_IOAT_AT` (`ACCESS_TOKEN`(250) ASC);
+
```
Comment (JKAUSHALYA) [inline en/docs/setup/wso2is_script.sql:73]: Remove extra new lines from the end.

**Comment by tharindu-bandara** [comment]
@SwatiNEC Would you be able to address the comments?

**Comment by gomathyk** [comment]
Closing this since the comments haven't been addressed. Feel free to re-open this PR after addressing given comments.

**Pull request closed without merging**